LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2004-2016 Castle Project - http://www.castleproject.org/
```

Comment:
Not sure how to read into your comment about noticing. I consciously chose to copy it as we still have a comprehensive clean up to do in a separate PR. So I did not think it was a problem.